components/patient-data/ui/src/main/resources/PhenoTips/AllergiesDataClass.xml

```diff
@@ -72,7 +72,7 @@
   #if($xcontext.action == 'edit' || $xcontext.action == 'inline')
     {{html wiki=false clean=false}}&lt;input class="xwiki-free-multiselect-value" type="text" size="30" name="${prefix}${name}" id="${prefix}${name}_${counter}" value="$!escapetool.xml($v)"/&gt;{{/html}}##
   #else
-    $escapetool.xml($v)##
+    $!{services.rendering.escape($v, $doc.syntax)}##
```

Comment:
Another question - why is .vm file on disk different from other velocity code?

And maybe we can create our own custom escape tool to avoid calling two different tools, which some developers may (which means "will")  forget to do?